services/storage/docker-compose.yml

```diff
@@ -55,7 +57,9 @@ services:
     stop_signal: SIGKILL
     env_file: [ ".env", ".storage.env", ".int_test.env" ]
     environment:
-      - NEOFS_NODE_KEY=/02.key
+      - NEOFS_NODE_WALLET_PATH=./wallet02.json
+      - NEOFS_NODE_WALLET_ADDRESS=NVXXy3hNTvwVEZa2dAibALyJB3Q86aiHvL
```

Comment:
Isn't that a redundant option? We only have one address in each wallet. Setting it here may complicate changing key in the future.

Reply:
As far as I know, neofs-node does not support default wallet addresses.

Reply:
We should support them, though https://github.com/nspcc-dev/neofs-node/blob/2da8396a9fce37c5df10f9e63e2d6807f77f72c3/cmd/neofs-cli/internal/key/wallet.go#L30

Reply:
Oh, sorry, this is node, not cli.

Reply:
Ok. We need not forget removing this when `neofs-node` gets default address support.